src/overlays/actors/ovl_Bg_Jya_Cobra/z_bg_jya_cobra.c

```diff
@@ -164,27 +164,19 @@ void BgJyaCobra_SpawnRay(BgJyaCobra* this, GlobalContext* globalCtx) {
     }
 }
 
-#ifdef NON_MATCHING
-// minor register and stack diffs
 void func_80895A70(BgJyaCobra* this) {
     s32 pad;
-    BgJyaBigmirror* temp_v1;
+    BgJyaBigmirror* temp_v1 = (BgJyaBigmirror*)this->dyna.actor.parent;
```

Comment:
```suggestion
    BgJyaBigmirror* mirror = (BgJyaBigmirror*)this->dyna.actor.parent;
```
Since there are multiple actors in here it might be helpful to name them.